bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup/ElectionKeyPair.cs

```diff
@@ -11,11 +11,15 @@ public class ElectionKeyPair : DisposableBase, IElectionGuardian
     /// The id of the owner guardian
     /// </summary>
     [Obsolete($"use {nameof(IElectionGuardian.GuardianId)}")]
-    public string OwnerId { get; set; }
+    public string OwnerId
+    {
+        get => GuardianId;
+        set => GuardianId = value;
+    }
 
-    // TODO: refactor to use IElectionGuardian
-    public string ObjectId => OwnerId;
-    public string GuardianId => OwnerId;
+    public string ObjectId => GuardianId;
+
+    public string GuardianId { get; set; }
```

Comment:
refactored to use the IElectionGuardian and inverted owner to use guardian instead of the other way around